### PR TITLE
[issue-376] Code clean up Task

### PR DESCRIPTION
**Pull request by ManoharVanam**
Removed TODO comments

### CODE REVIEW

File: core/src/main/java/org/carbondata/query/cache/QueryExecutorUtil.java [hunk 1]
```diff
@@ -22,7 +22,6 @@
 import java.util.Arrays;
```
Comment (kumarvishal09) [inline core/src/main/java/org/carbondata/query/cache/QueryExecutorUtil.java]: remove this class

**Comment by ravipesala** [comment]
Please rebase it

**Comment by jackylk** [comment]
Do these code going to be removed include the logic for aggregated table which we are going to add later?

**Comment by ManoharVanam** [comment]
These changes are not required

**Pull request closed without merging**